src/Api/SecretsManager/Models/Response/SMImportResponseModel.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Bit.Core.Models.Api;
+using Bit.Core.SecretsManager.Commands.Porting;
+
+namespace Bit.Api.SecretsManager.Models.Response;
+
+public class SMImportResponseModel : ResponseModel
+{
+    public SMImportResponseModel(SMImport import, string obj = "SecretsManagerImportResponseModel") : base(obj)
+    {
+        Projects = import.Projects != null && import.Projects.Any() ? import.Projects.Select(p => new InnerProject
+        {
+            Id = p.Id,
+            Name = p.Name,
+        }) : null;
+
+        Secrets = import.Secrets != null && import.Secrets.Any() ? import.Secrets.Select(s => new InnerSecret
+        {
+            Id = s.Id,
+            Key = s.Key,
+            Value = s.Value,
+            Note = s.Note,
+            ProjectIds = s.ProjectIds,
+        }) : null;
+    }
+
+    public IEnumerable<InnerProject> Projects { get; set; }
+    public IEnumerable<InnerSecret> Secrets { get; set; }
+
+    public class InnerProject
```

Comment:
Swagger breaks if we have identical classnames. Can we rename these to contain import? Perhaps `InnerProjectImportResponseModel`.

Reply:
Updated!